wscript

```diff
@@ -143,12 +143,12 @@ def options(opt):
 
     g.add_option('--Werror',
         action='store_true',
-        default=False,
+        default=None,
```

Comment:
The change to use `None` make a lot more sense in the logic because it was impossible to tell if the CLI overrode the default or not. Great change Peter.